docs/eventing/sources/apiserversource.md

```diff
@@ -0,0 +1,267 @@
+         image: gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display
+EOF
+```
+
+### Create Events
+
+Create events by launching a pod in the default namespace. Create a `busybox`
+container and immediately delete it:
+
+```shell
+kubectl -n apiserversource-example run busybox --image=busybox --restart=Never -- ls
+kubectl -n apiserversource-example delete pod busybox
+```
+
+### Verify
+
+We will verify that the Kubernetes events were sent into the Knative eventing
+system by looking at our message dumper function logs. If you deployed the
+[Trigger](#trigger), continue using this section. If not, you will need to look
+downstream yourself:
+
+```shell
+kubectl -n apiserversource-example get pods
+kubectl -n apiserversource-example logs -l serving.knative.dev/service=event-display -c user-container --tail=200
+```
+
+You should see log lines similar to:
+
+```
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: dev.knative.apiserver.resource.update
+  source: https://10.96.0.1:443
+  subject: /apis/v1/namespaces/apiserversource-example/events/testevents.15dd3050eb1e6f50
+  id: e0447eb7-36b5-443b-9d37-faf4fe5c62f0
+  time: 2020-07-28T19:14:54.719501054Z
+  datacontenttype: application/json
+Extensions,
+  kind: Event
+  knativearrivaltime: 2020-07-28T19:14:54.720059098Z
+  knativehistory: default-kne-trigger-kn-channel.apiserversource-example.svc.cluster.local
+  name: busybox.1626008649e617e3
+  namespace: apiserversource-example
+Data,
+  {
+    "apiVersion": "v1",
+    "count": 1,
+    "eventTime": null,
+    "firstTimestamp": "2020-07-28T19:14:54Z",
+    "involvedObject": {
+      "apiVersion": "v1",
+      "fieldPath": "spec.containers{busybox}",
+      "kind": "Pod",
+      "name": "busybox",
+      "namespace": "apiserversource-example",
+      "resourceVersion": "28987493",
+      "uid": "1efb342a-737b-11e9-a6c5-42010a8a00ed"
+    },
+    "kind": "Event",
+    "lastTimestamp": "2020-07-28T19:14:54Z",
+    "message": "Started container",
+    "metadata": {
+      "creationTimestamp": "2020-07-28T19:14:54Z",
+      "name": "busybox.1626008649e617e3",
+      "namespace": "default",
+      "resourceVersion": "506088",
+    "selfLink": "/api/v1/namespaces/apiserversource-example/events/busybox.1626008649e617e3",
+      "uid": "2005af47-737b-11e9-a6c5-42010a8a00ed"
+    },
+    "reason": "Started",
+    "reportingComponent": "",
+    "reportingInstance": "",
+    "source": {
+      "component": "kubelet",
+      "host": "gke-knative-auto-cluster-default-pool-23c23c4f-xdj0"
+    },
+    "type": "Normal"
+  }
+```
+
+### Cleanup
+
+Delete the `apiserversource-example` namespace and all of its resources from your
+cluster by entering the following command:
+
+```shell
+kubectl delete namespace apiserversource-example
+```
+
+## Reference Documentation
+
+See the [APIServerSource specification](../../reference/eventing/#sources.knative.dev/v1alpha2.APIServerSource).
```

Comment:
v1beta1?

Reply:
Sounds good. Initially I thought we haven't changed the storage version yet, so that could be v1alpha2.